### ISSUE TITLE
local variables for kwargs not inlined

### ISSUE THREAD

**Issue by wkeese** [csnover/js-doc-parse#16]
For dijit/form/_CurrencyTextBox, __Constraints is defined as a local variable:

``` javascript
/*=====
var __Constraints = declare([NumberTextBox.__Constraints, currency.__FormatOptions, currency.__ParseOptions], {
    // summary:
    //      Specifies both the rules on valid/invalid values (minimum, maximum,
    //      number of required decimal places), and also formatting options for
    //      displaying the value when the field is not focused (currency symbol,
    //      etc.)
    // description:
    //      Follows the pattern of `dijit.form.NumberTextBox.constraints`.
    //      In general developers won't need to set this parameter
    // example:
    //      To ensure that the user types in the cents (for example, 1.00 instead of just 1):
    //  |       {fractional:true}
});
=====*/
```

Therefore, when that type definition is used:

``` javascript
/*=====
// constraints: __Constraints
//      Despite the name, this parameter specifies both constraints on the input
//      (including minimum/maximum allowed values) as well as
//      formatting options.
constraints: {},
======*/
```

It's supposed to be inlined into a  UL LI ... type list in the description, right?   It's not:

``` xml
<property name="constraints" scope="prototype" type="__Constraints" from="dijit/form/CurrencyTextBox">
<summary>
Despite the name, this parameter specifies both constraints on the input (including minimum/maximum allowed values)
 as well as formatting options.
</summary>
</property>
```

**Comment by csnover** [comment]
Fixed in d6b0fe1. One of these days I will remember to directly reference the issue number in the commit message…

**Comment by wkeese** [comment]
Cool, it's working for me.
